src/validations/rules/ssp.sch

```diff
@@ -1645,8 +1645,12 @@
                 diagnostics="uses-correct-param-descriptions-diagnostic"
                 id="uses-correct-param-descriptions"
                 role="error"
-                test="$profile-catalog-control-params/array:controls/o:control/o:param[@id eq $paramID][o:label eq $paramValue or o:constraint/o:description/o:p eq $paramValue]">A
-                FedRAMP SSP must use correct parameter descriptions.</sch:assert>
+                test="
+                    if ($profile-catalog-control-params/array:controls/o:control/o:param[@id eq $paramID]/o:constraint)
+                    then
+                        ($profile-catalog-control-params/array:controls/o:control/o:param[@id eq $paramID][o:constraint/o:description/o:p eq $paramValue])
+                    else
+                        (true())">A FedRAMP SSP must use correct parameter descriptions.</sch:assert>
```

Comment:
Shouldn't the message (and related identifiers) say something to the effect that a parameter **value** (`set-parameter/value`) **value** matches the related constraint? The assertion messages (assert and diagnostic) refer to **descriptions**, not **values**.